src/js/components/award/contract/ContractDetails.jsx

```diff
@@ -100,12 +102,19 @@ export default class ContractDetails extends React.Component {
             pricing = award.type_of_contract_pricing_description;
         }
 
+        // char count
+        let seeMore = false;
+        if (award.description.length > 160) {
```

Comment:
Let's move this magical `160` number into class property and pass it down to `DetailRow`